flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/operators/over/NonTimeRangeUnboundedPrecedingFunction.java

```diff
@@ -422,11 +422,13 @@ private Tuple2<Integer, Boolean> findIndexOfSortKey(
     private void setAccumulatorOfPrevRow(
             List<Tuple2<RowData, List<Long>>> sortedList, int prevIndex) throws Exception {
         if (prevIndex < 0) {
-            aggFuncs.createAccumulators();
+            RowData accData = aggFuncs.createAccumulators();
+            aggFuncs.setAccumulators(accData);
```

Comment:
```suggestion
            aggFuncs.setAccumulators(aggFuncs.createAccumulators());
```
nest the calls? the accumulator object is not needed.